modules/openfast-library/src/FAST_Solver.f90

```diff
@@ -582,6 +654,55 @@ SUBROUTINE AD_InputSolve_IfW( p_FAST, u_AD, y_IfW, y_ExtInfw, ErrStat, ErrMsg )
    
 END SUBROUTINE AD_InputSolve_IfW
 !----------------------------------------------------------------------------------------------------------------------------------
+
+SUBROUTINE AD_InputSolve_IfW_ExtLoads( p_FAST, u_AD, p_ExtLd, ErrStat, ErrMsg )
+
+  type(FAST_ParameterType), intent(in)   :: p_FAST     !< FAST parameter data
+  type(AD_InputType), intent(inout)      :: u_AD       !< The inputs to AeroDyn
+  type(ExtLd_ParameterType), intent(in)  :: p_ExtLd    !< Parameters of ExtLoads
+  integer(IntKi)                         :: ErrStat    !< Error status of the operation
+  character(*)                           :: ErrMsg     !< Error message if ErrStat /= ErrID_None
+
+  !local variables
+  real(ReKi)                             :: z          !< Local 'z' coordinate
+  real(ReKi)                             :: mean_vel   !< Local mean velocity
+  real(ReKi)                             :: pi         !< Our favorite number
+  integer(IntKi)                         :: j,k        !< Local counter variables
+  integer(IntKi)                         :: NumBl      !< Number of blades
+  integer(IntKi)                         :: Nnodes     !< Number of nodes
+
+  ErrStat = ErrID_None
+  ErrMsg = ''
+  
+  pi = acos(-1.0)
+  NumBl  = size(u_AD%rotors(1)%InflowOnBlade,3)
+  Nnodes = size(u_AD%rotors(1)%InflowOnBlade,2)
+
+  do k=1,NumBl
+     do j=1,Nnodes
+        !Get position first
+        z = u_AD%rotors(1)%BladeMotion(k)%Position(3,j) + u_AD%rotors(1)%BladeMotion(k)%TranslationDisp(3,j)
```

Comment:
This will need revision with `dev-unstable-pointers` when the inflow is no longer passed.